release/charts/charts.go

```diff
@@ -66,6 +102,60 @@ func Update(ctx context.Context, c *config.ChartsRelease, br, ch, vr string) (st
 	return string(output), nil
 }
 
+// Push will push the charts updates to the remote upstream charts repository
+func Push(ctx context.Context, c *config.ChartsRelease, u *config.User, ghc *github.Client, b, t string, d bool) (string, error) {
```

Comment:
Can you add more context for `b`, `t` and `d` on the function docs above?

Reply:
Done